scripts/mkcloud

```diff
@@ -1236,6 +1236,8 @@ Steps:
     addupdaterepo:  addupdate repos defined in UPDATEREPOS= (URLs separated by '+')
     runupdate:      run zypper up on the crowbar node
                     (compute nodes are automatically updated via chef run)
+    cloudupgrade:   upgrade the cloud to the version specified
+                    by the `upgrade_cloudsource` parameter
```

Comment:
this is a shell script. the backticks call the command inside.

Reply:
Could you please suggest how the text should look like?

Reply:
just remove the backticks :o)